clustering/ejb/WFLY-7628_Distributed_EJB_Timers.adoc

```diff
@@ -56,87 +59,108 @@ WildFly needs an persistent timer service implementation suitable for use in a c
 
 === Non-Requirements
 
+* This feature will not change the default timer service behavior or deprecate any existing timer service mechanisms.
+
 == Backwards Compatibility
 
-* Servers started using an old version of the EJB subsystem configuration will continue to behave as normal, using the existing TimerServiceImpl for both transient and persistent timers.
+* Servers started using the default ejb3 subsystem configuration will continue to behave as normal, using the existing in-memory `TimerService` implementation for both transient and persistent timers.
 
 === Default Configuration
 
-The following caches will be added to the default ejb cache-container configuration with the Infinispan subsystem:
+The following caches will be added to the default `ejb` cache-container configuration with the Infinispan subsystem:
 
 For the HA profiles:
 
-	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent-timers:add()
-	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent-timers/component=locking:add(isolation=PESSIMISTIC)
-	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent-timers/component=transaction:add(mode=BATCH)
-	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent-timers/store=file:add(purge=false)
+	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent:add()
+	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent/component=locking:add(isolation=PESSIMISTIC)
+	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent/component=transaction:add(mode=BATCH)
+	/subsystem=infinispan/cache-container=ejb/distributed-cache=persistent/store=file:add(purge=false)
 
 And for non-HA profiles:
 
-	/subsystem=infinispan/cache-container=ejb/local-cache=persistent-timers:add()
-	/subsystem=infinispan/cache-container=ejb/local-cache=persistent-timers:component=locking:add(isolation=PESSIMISTIC)
-	/subsystem=infinispan/cache-container=ejb/local-cache=persistent-timers:component=transaction:add(mode=BATCH)
-	/subsystem=infinispan/cache-container=ejb/local-cache=persistent-timers/store=file:add(purge=false,passivation=false)
+	/subsystem=infinispan/cache-container=ejb/local-cache=persistent:add()
+	/subsystem=infinispan/cache-container=ejb/local-cache=persistent:component=locking:add(isolation=PESSIMISTIC)
+	/subsystem=infinispan/cache-container=ejb/local-cache=persistent:component=transaction:add(mode=BATCH)
+	/subsystem=infinispan/cache-container=ejb/local-cache=persistent/store=file:add(purge=false, passivation=false)
 
 The following timer management provider will be added to the default distributable-ejb subsystem configuration:
 
-	/subsystem=distributable-ejb/infinispan-timer-management=persistent:add(cache-container=ejb, cache=persistent-timers)
+	/subsystem=distributable-ejb/infinispan-timer-management=persistent:add(cache-container=ejb, cache=persistent)
```

Comment:
So from what I understand, what above will be added to the server default configuration, while what follows must be set by the user to enable the feature. Is this correct or am I missing something?

Reply:
Correct.

Reply:
Thanks for clarifying